AlembicImporter/Packages/com.unity.formats.alembic/Documentation~/timeline.md

```diff
@@ -0,0 +1,8 @@
+# Using Alembic with Timeline
+
+You can use Timeline to play back and record Alembic animation including:
+
+* Playing Alembic animation by [controlling the Alembic Stream Player component](time_InfiniteClip.html) through an Infinite clip. This takes a little longer to set up than using an Alembic clip, but you can easily convert an Infinite clip to an Animation clip to use in an Animator controller (state machine). 
```

Comment:
I think the below two methods are used more often, so maybe we should put this one last (?)

Reply:
Makes sense